### PR TITLE
PR for BXMSDOC-7311: Incorporating feedback on Managing and monitoring KIE Server document.

### PR DESCRIPTION
**Pull request by kaldesai**
**JIRA:** 
- [Epic](https://issues.redhat.com/browse/BXMSDOC-7311)

**Doc previews:**
- [RHPAM-Managing and monitoring KIE server](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-7656-MMKS-RHPAM/)
- [RHDM-Managing and monitoring KIE server](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-7656-MMKS-RHDM/)

**Doc impact:**
Check chapters **18.7. Pessimistic locking support** and **21. KIE Server system properties** for the updates in RHPAM doc.

### CODE REVIEW

**Comment by fjtirado** [comment]
Looks good to me, just change "interpret" by "check"

File: doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-system-properties-ref.adoc [hunk 1]
```diff
@@ -368,4 +368,10 @@ endif::[]
 |`true`, `false`
 |`false`
 |While using JSON marshalling, if the property is set to `true`, it will always return a response in the proper JSON format. For example, if the original response contains only a single number, then the response is wrapped in a JSON format. For example, *{"value" : 1}*.
+
+|`org.kie.server.strict.jaxb.format`
+|`true`, `false`
+|`false`
+|When the property is set to `true`, the {KIE_SERVER} will interpret data in the REST API payload. For example, if a data field is supposed to be a number but it is not a number, you receive an error.
```
Comment (briandooley) [inline doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-system-properties-ref.adoc:375]:
```suggestion
|When set to `true`, the {KIE_SERVER} interprets the data in the REST API payload. For example, if a data field is supposed to be a number, but it is not a number, you receive an error.
```
I noticed that similar text in this table says "when set to true" and not "When the property is set to true" so I suggest bringing it in line with them. Suggestion also removes passive voice.
"will interpret" - p77 of the style guide. "Write in the simple present tense as much as possible.".

Also, should "interprets" be "validates"?

**Review by briandooley — CHANGES_REQUESTED** [review]

File: doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-system-properties-ref.adoc (hunk 1 above)
Comment (emmurphy1) [reply doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-system-properties-ref.adoc:375]: Great comments, Brian. I suggest:
When the value of this property is set to 'true' and do the same for the other one in the table.
It's always KIE_SERVER, not the KIE_SERVER when KIE_SERVER is a noun. You can use 'the' when KIE_SERVER is an adjective. For example, configure the KIE_SERVER instance.

I suggest:
When the value of this property is set to `true`, {KIE_SERVER} validates the data type of the data in the REST API payload. For example, if a data field has the number data type and contains something other than a number, you will receive an error.

Kalyani, confirm this with your SME.

Comment (kaldesai) [reply doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-system-properties-ref.adoc:375]: @briandooley, @emmurphy1
Thank you so much for the suggestions. Yes, I confirmed this with the SME and acked it.

File: doc-content/enterprise-only/admin-and-config/pessimistic-locking-cli-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
+[id='pessimistic-locking-cli-proc_{context}']
+= Configuring pessimistic locking support using CLI mode
```
Comment (MarianMacik) [inline doc-content/enterprise-only/admin-and-config/pessimistic-locking-cli-proc.adoc:2]: I wouldn't call it a CLI mode as we don't really have a Command Line Interface. Can you rename it to just `Configuring pessimistic locking support programmatically`?

**Pull request merged**